Makefile

```diff
@@ -16,87 +12,147 @@ endif
 # You can use colors like this:
 # @echo "${RED}RED${RESET}"
 
-### Easy to use Docker commands ###
+### Encrypt and decrypt commands for secret variables (call from d_prod_install) ###
+ENVPROD = .envs/.prod/.env.prod
+ENVPRODTMP = $(ENVPROD).tmp
+ENVPROD_DB = .envs/.prod/.env.prod.db
+ENVPRODTMP_DB = $(ENVPROD_DB).tmp
+decrypt_env:
+	@echo "${GREEN}[#] DECRYPTING ENV FILES"
+	sops --decrypt $(ENVPROD) >> $(ENVPRODTMP) && rm $(ENVPROD) && mv $(ENVPRODTMP) $(ENVPROD)
+	sops --decrypt $(ENVPROD_DB) >> $(ENVPRODTMP_DB) && rm $(ENVPROD_DB) && mv $(ENVPRODTMP_DB) $(ENVPROD_DB)
+	@echo "-- ${RESET}"
 
+GCPKEY = projects/victor-ciurana-com/locations/global/keyRings/victorciurana-keyring/cryptoKeys/victorciuranacom-key
+encrypt_env:
+	@echo "${GREEN}[#] ENCRYPTING ENV FILES"
+	sops --encrypt --gcp-kms $(GCPKEY) $(ENVPROD) > $(ENVPRODTMP) && rm $(ENVPROD) && mv $(ENVPRODTMP) $(ENVPROD)
+	sops --encrypt --gcp-kms $(GCPKEY) $(ENVPROD_DB) > $(ENVPRODTMP_DB) && rm $(ENVPROD_DB) && mv $(ENVPRODTMP_DB) $(ENVPROD_DB)
+	@echo "-- ${RESET}"
```

Comment:
You shouldn't need top encrypt anything manually, when you want to edit a secrets file, just do `sops .prod.env` for instance, and edit the values. Then sops automatically encrypts it when you are finished editing

Reply:
You are right I knew I could not be that complicated